src/components/policy/src/cache_manager.cc

```diff
@@ -1187,6 +1187,10 @@ bool CacheManager::IsPredataPolicy(const std::string& app_id) {
       pt_->policy_table.app_policies_section.apps[app_id];
 
   policy_table::Strings res;
+
+  sort(pre_data_app.groups.begin(), pre_data_app.groups.end());
```

Comment:
@LuxoftAKutsan fixed in [3d9672a](https://github.com/smartdevicelink/sdl_core/pull/794/commits/3d9672a458e8d7f73abde3ce47b8005395cbd63e)